server/scsynth/SC_World.cpp

```diff
@@ -55,6 +55,13 @@
 
 #include "malloc_aligned.hpp"
 
+#include <boost/predef/architecture.h>
+#include <boost/predef/hardware.h>
+
+#ifdef BOOST_HW_SIMD_AVAILABLE
```

Comment:
should be `BOOST_HW_SIMD_X86 >= BOOST_HW_SIMD_X86_SSE_VERSION` (BOOST_HW_SIMD_AVAILABLE is for all SIMD sets, but `xmmintrin.h` is x86-specific)